SafeTrade/__init__.py

```diff
@@ -11,7 +11,7 @@
 from SafeTrade.database.MongoDB import check_mongo_uri
 from SafeTrade.database.Redis import OrderHandler
 
-uvloop.install()
```

Comment:
this too

Reply:
i`wont
i just dont know how to commit just the first commit not WinDev
if its not possible I assume I should just don't push them anywhere